src/styles/_variables.scss

```diff
@@ -1,8 +1,14 @@
+:root {
```

Comment:
New css variables. Need to move this someplace else because is getting injected in ALL the css compiled code.